dev/source/docs/how-the-team-works-voting.rst

```diff
@@ -42,9 +42,9 @@ Description belows how to creat a form from scratch, alternatively you can use `
   - set the "Email To" field to "ardupilot-dev-team@googlegroups.com"
   - fill in the Subject and Message fields
   - to allow others to see the responses (including ardupilot.devel@gmail.com), click the "Collaborators" link and add their email in the "Invite people" field.  If you are submitting the vote as yourself, include 
-  - check "Include form in email" and press "Send"
+  - check "Include form in email" and press "Send".
 
-Normally the vote should be open for 1 week and be discussed at the weekly development call.
+The votes are generally open for 1 week and will be discussed at the weekly development call.
```

Comment:
The original english is correct actually.  The replacement is also correct but the change doesn't improve the english I think.